tripal/src/TripalDBX/TripalDbxSchema.php

```diff
@@ -1079,6 +1080,54 @@ public function getReferencingTables(
     return $db_dependencies[$cache_key];
   }
 
+ /**
```

Comment:
Coding standards changes:
```suggestion
  /**
```